EIPS/eip-5252.md

```diff
@@ -0,0 +1,237 @@
+---
+eip: 5252
+title: Account-bound Finance
+description: An EIP-5114 extension that aids in preventing arbitrary loss of funds
+author: Hyungsuk Kang (@hskang9), Viktor Pernjek (@smuxx)
+discussions-to: https://ethereum-magicians.org/t/pr-5252-discussion-account-bound-finance/10027
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-06-29
+requires: 20, 721, 1155, 5114
+---
+
+## Abstract
+This EIP proposes a form of smart contract design pattern and a new type of account abstraction on how one's finance should be managed in web3.0, ensuring transparency of managing investments and protection with self-sovereignty even from its financial operators.
+
+This EIP extends [EIP-5114](./eip-5114.md) to further enable transferring fund to other accounts for mobility between managing multiple wallets(i.e. private keys).
+
+## Motivation
+
+Finance in the crypto ecosystem is facing a huge trust issue. Smart contracts are often proxies, with the actual logic of the contract hidden away in a separate logic contract. Many projects include a multi-signature "wallet" with unnecessarily-powerful permissions. And it is not possible to independently verify that stablecoins have enough real-world assets to continue maintaining their peg, creating a large loss of funds (such as happened in the official bankruptcy announcement of Celsius and UST de-pegging and anchor protocol failure). One should not trust exchanges or other third parties with one's own investments with the operators' background, status, or cult of personality in web3.0.
+
+The pattern empowers more self-sovereignty and gives more credentials(obviously with an Account-bound token) than locking financial data in the operating team's contract with a trustless way.
```

Comment:
Required: Rephrase this. It is unclear what you are trying to say.

Reply:
I was stating, as a fact, that what you are saying was (and is still) unclear to me, and so needs rephrasing. I will not, as per EIP-1, merge this draft until I can actually fully understand what is being said here. All I'm looking for here is for you to elaborate - how is an ABT the best way to empower greater self-sovereignty? What do you mean by "give more credentials"? How does one "[lock] financial data"? This is currently unclear to me as written, and hopefully should be easy for you to fix. If not, then you can always remove it and re-add it in a later revision.

As for your other point, I am indeed editing your proposal. I am admittedly being blunt and trite here (I find that it oftentimes means that lengthy discussions over minutiae are avoided, but clearly not this time), but all my feedback is constructive, targeted, and actionable. I would be more than willing to stop editing your proposal if you so wish, but I imagine that both you and I would rather get this EIP merged. In order to expedite that, please make the above change.